docs/source/apps/projsync.rst

```diff
@@ -0,0 +1,158 @@
+.. _projsync:
+
+================================================================================
+projsync
+================================================================================
+
+.. Index:: projsync
+
+.. only:: html
+
+    .. versionadded:: 7.0.0
+
+    Tool for downloading resource files
```

Comment:
Tool for synchronizing PROJ datum and transformation support data.

Reply:
done